l1-contracts/upgrade-envs/v0.27.1-gateway-activation/mainnet.toml

```diff
@@ -0,0 +1,58 @@
+era_chain_id = 324
+testnet_verifier = false
+governance_upgrade_timer_initial_delay = 1058400
```

Comment:
This seems like 12 days, I think it is a bit too much. Let's put 2-3 days there or something like that.